ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -543,14 +558,21 @@ CMitsubaLoader::SContext::shape_ass_type CMitsubaLoader::loadBasicShape(SContext
 						auto meshbuffer = mesh->getMeshBuffer(i);
 						uint32_t offset = reinterpret_cast<uint8_t*>(it)-reinterpret_cast<uint8_t*>(newRGB->getPointer());
 						core::vectorSIMDf rgb;
-						for (uint32_t i=0u; meshbuffer->getAttribute(rgb, asset::EVAI_ATTR1, i); i++,it++) // should be upstreamed into the PLY loader
+						for (uint32_t i=0u; meshbuffer->getAttribute(rgb, 1u, i); i++,it++) // should be upstreamed into the PLY loader
 						{
 							for (auto i=0; i<3u; i++)
 								rgb[i] = video::impl::srgb2lin(rgb[i]);
 							meshbuffer->setAttribute(rgb,it,asset::EF_A2B10G10R10_UNORM_PACK32);
 						}
-						meshbuffer->getMeshDataAndFormat()->setVertexAttrBuffer(
-								core::smart_refctd_ptr(newRGB),asset::EVAI_ATTR1,asset::EF_A2B10G10R10_UNORM_PACK32,hidefRGBSize,offset);
+						constexpr uint32_t COLOR_BUF_BINDING = 15u;
```

Comment:
why at the last binding?

Reply:
it's kinda dirty (not 100% safe but very unlikely to be unsafe), i'm assuming that binding 15 is not used by PLY loader

Reply:
what is this color buffer even being used for?